src/main/scad/yavuz-demirhan/Raya1.scad

```diff
@@ -0,0 +1,66 @@
+include <puzzlecad2.scad>
+
+box_puzzle_scale = 15;
+box_puzzle_border = 6;
+$burr_inset = 0.16; 
+long = box_puzzle_scale * 4; // voxels long
+wide = box_puzzle_scale * 3; // voxels wide
+high = box_puzzle_scale * 2; // voxels high
+
+dimlong = long + box_puzzle_border * 2;
+dimwide = wide + box_puzzle_border * 2;
+height = high + box_puzzle_border;
+
+mid = box_puzzle_border / 2 + 0.5;
+far = dimlong - mid;
+far2 = dimwide - mid;
+
+pieces();
+*box();
+*cap();
+
+module pieces() {
+    burr_plate([
+    ["xx|xx"],  //piece A
+    ["x..|xxx"],//piece B
```

Comment:
I'd suggest we explicitly declare multiple copies of each piece, to make it easier for users to print the right number of copies (especially since the distribution of pieces varies across the different Raya puzzles).

Reply:
Agree.